corehq/messaging/scheduling/templates/scheduling/partials/conditional_alert_save.html

```diff
@@ -1,7 +1,27 @@
 {% load i18n %}
 <div class="form-actions">
+
+  <div class="modal fade" id="conditional-alert-warning-modal" role="dialog">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h4 class="modal-title">Conditional Alert</h4>
+        </div>
+        <div class="modal-body">
+          <p>This update may alert <span id="case-count"></span> cases. Are you sure you want to proceed?</p>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal" id="conditional-alert-yes-btn">Yes</button>
+          <button type="button" class="btn btn-default" data-dismiss="modal" id="conditional-alert-no-btn">No</button>
```

Comment:
This page uses a lot of ids because it's built with crispy forms, which automatically adds an id to every form element. But for HTML we're writing ourselves, there's no need to add ids to elements if those ids aren't going to be used.